tree/dataframe/src/RDFInterfaceUtils.cxx

```diff
@@ -847,6 +847,19 @@ std::vector<bool> FindUndefinedDSColumns(const ColumnNames_t &requestedCols, con
    return mustBeDefined;
 }
 
+void CheckForDuplicateSnapshotColumns(const ColumnNames_t &cols)
+{
+   std::set<std::string> uniqueCols;
```

Comment:
```suggestion
   std::unordered_set<std::string> uniqueCols;
```
?

Reply:
okaaaay, for the >20 columns cases https://quick-bench.com/q/GVs0W7VRxV1xSp48zOsiFZNcCJI

Reply:
Done